src/Component.php

```diff
@@ -14,9 +14,23 @@ class Component extends BaseComponent
 {
     private const CHECK_ACTION = 'check';
 
-    protected function run(): void
+    private const CHECK_MIGRATE = 'migrate';
+
+    protected function migrate(): array
     {
-        // @TODO implement
+        $application = new Application(
+            $this->getConfig(),
+            new Components($this->getStorageClient())
+        );
+
+        if ($this->getConfig()->hasTransformationId()) {
```

Comment:
`transformationId` je `required` v `ConfigDefinition`, tak `has` metodu by si nemal vobec potrebovat?

Reply:
no úplně to potřeba není, ošetřuje to config na vstupu:
```
{
  "action": "migrate"
}
```